### PR TITLE
Adding other meetups in the Newcastle Area

### PR DESCRIPTION
**Pull request by KleeUT**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -279,7 +279,74 @@
...
+        <a href="https://www.meetup.com/Newcastle-Infracoders/">Newcastle Infracoders and DevOps</a>
+      </h4>
+      <dl>
+        <dt>Links:</dt>
+        <dd>
+          <ul>
+            <li><a href="https://www.meetup.com/Newcastle-Infracoders/" rel="external">Meetup</a></li>
+          </ul>
+        </dd>
+        <dt>When:</dt>
```
Comment (neozenith) [inline index.html]: There are no details as yet about regular times for infracoders?

File: index.html [hunk 2]
```diff
@@ -279,7 +279,74 @@
...
+        <a href="https://www.meetup.com/Agile-Newcastle/">Agile Newcastle</a>
+      </h4>
+      <dl>
+        <dt>Links:</dt>
+        <dd>
+          <ul>
+            <li><a href="https://www.meetup.com/Agile-Newcastle/" rel="external">Meetup</a></li>
+          </ul>
+        </dd>
+        <dt>When:</dt>
```
Comment (neozenith) [inline index.html]: No regular details as yet? Perhaps mark these TBA

File: index.html (hunk 1 above)
Comment (KleeUT) [inline index.html]: This group meets erratically.  I think the schedule is defined by the availability of speakers.

File: index.html (hunk 2 above)
Comment (KleeUT) [inline index.html]: This group meets erratically.  I think the schedule is defined by the availability of speakers.

**Comment by neozenith** [comment]
:shipit:

**Pull request merged**